skaffold.yaml

```diff
@@ -89,7 +89,7 @@ portForward:
   resourceName: hydra-public
   namespace: default
   port: 4444
-  localPort: 14444
+  localPort: 4444
```

Comment:
question: what is the problem with the current configuration? I thought that we weren't going to use hydra as a provider. 

If we want to enable hydra as a provider, why don't we change the issuer_url to point to port `14444` instead to keep consistency with how we expose apps via skaffold?

Reply:
The problem is that the local setup doesn't allow the app to work, and while developing I need to have a quick way to try things without relying on Canonical infra.

The specific problem is the issuer validation, which doesn't match the one queried when the response is returned from the oidc discovery, causing the startup to fail (it would still fail validation on tokens "later" in the app life).

I changed it here because the skaffold deployment was the "broken" one, meanwhile the docker compose one was not.
If we keep the 14444 port here, we would have to change it in the docker compose (breaking consistency over there as well).
We could just adopt the same approach for all ports in both of our local deployment setups. My main focus here was fixing things so it works.
I'm open to whatever in this case